cl/corpus_importer/tests.py

```diff
@@ -311,14 +313,23 @@ class PacerDocketParserTest(TestCase):
     NUM_FLOYD_ROLES = 3
     NUM_DOCKET_ENTRIES = 123
 
-    def setUp(self) -> None:
+    @classmethod
+    def setUpTestData(cls) -> None:
         docket_number = "3:11-cv-00064"
-        self.docket = find_docket_object("akd", "41664", docket_number)
-        self.docket.filepath_local = (
-            "/test/xml/gov.uscourts.akd.41664.docket.xml"
+        cls.docket = DocketFactory.create(
+            source=Docket.RECAP,
+            pacer_case_id="41664",
+            docket_number=docket_number,
+            court=Court.objects.get(id="akd"),
```

Comment:
We want to remove the fixture on line 309, right? And use the CourtFactory here to create the akd court?

Reply:
√

Reply:
Interesting, so it looks like we get `akd` from court_data_truncated.json. Do you think you could take a stab at removing `akd` from that fixture? I just took a quick look and it only seems to affect a few other classes, I think. Eventually we'll remove the court_data_truncated.json fixture completely and get a nice speed boost, but this is a good opportunity to chip away at it while we're on the topic.